dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParamsMapperTest.java

```diff
@@ -483,6 +481,8 @@ void shouldFailWhenOrgUnitSuppliedAndOrgUnitModeAccessible() {
     requestParams.setOrgUnit(UID.of(orgUnit.getUid()));
     requestParams.setOrgUnitMode(ACCESSIBLE);
 
+    when(organisationUnitService.getOrganisationUnit(orgUnit.getUid())).thenReturn(orgUnit);
```

Comment:
nit: I think we put the parameter to the method/function we test close to the call of the tested method. So the mock would go above the `requestParams` statements.

Reply:
my memory is really bad lately, so I could have easily have forgotten it, when did we decide this?

Reply:
I don't think we made a decision. Its just what I see us doing in all the mapper tests AFAIK 😅